src/main/scala/Domain/Door.scala

```diff
@@ -56,7 +56,7 @@ object Door {
       }
 
   val aggregateLogic: List[Flow[Unit]] = List(
-    handler { case Register(id) => emitEvent(Registered(id)) } >> waitFor { case Registered(_) => () },
+    handler { handler[Register, Registered] } >> waitFor { case Registered(_) => () },
```

Comment:
on another hand, `noValidation` is not a good idea - as the eventflow validates that the aggregate is in an acceptable state for the handler already :)